app/utils/__tests__/createEntityReducer.spec.js

```diff
@@ -382,3 +408,168 @@ describe('entities()', () => {
     });
   });
 });
+
+describe('deleteEntities()', () => {
+  it('should delete the given entity', () => {
+    const reducer = deleteEntities(DELETE, 'events');
+
+    const state = {
+      actionGrant: [],
+      byId: {
+        1: {
+          id: 1,
+          title: 'Second Event',
+          comments: []
+        },
+        2: {
+          id: 2,
+          title: 'Second Event',
+          comments: []
+        }
+      },
+      items: [1, 2],
+      pagination: {}
+    };
+
+    const action = {
+      type: 'DELETE_SUCCESS',
```

Comment:
Use DELETE.SUCCESS?

Reply:
Just kept the same as in the other tests, but I guess we can switch to `DELETE.SUCCESS`